layout/_partials/head/head.swig

```diff
@@ -1,6 +1,6 @@
 <meta charset="UTF-8"/>
 <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2"/>
```

Comment:
This will not to do site larger by double click?

Reply:
No.
[here](https://dequeuniversity.com/rules/axe/2.2/meta-viewport?application=lighthouse) said should disable zoom

Reply:
but in wechat, a zoom buttom in **lower left corner** appears. could it be optional in _config.yml? @maple3142 
![374975667](https://user-images.githubusercontent.com/16632254/41760264-2fda052a-7625-11e8-8e24-cac58a2a82ab.jpg)